nuxeo-runtime/nuxeo-launcher-commons/src/test/resources/configurator/nuxeo.conf

```diff
@@ -21,3 +21,12 @@ nuxeo.wizard.done=false
 test.sampled.prop2=someValue
 
 nuxeo.force.generation=true
+
+nuxeo.db.host=${env:NUXEO_DB_HOST}
+
+org.nuxeo.fake.vindoz=${env?:NUXEO_UNKNOWN_VAR}
```

Comment:
I don't see any test checking the value of `org.nuxeo.fake.vindoz`. You should have test cases for true and false btw.